R/tm_a_gee.R

```diff
@@ -154,6 +154,9 @@ template_a_gee <- function(output_table,
 #' ```
 #'
 #' For additional details and examples of decorators, refer to the vignette
+#' `vignette("decorate-module-output, package = "teal.modules.clinical")`.
```

Comment:
```suggestion
#' `vignette("decorate-module-output", package = "teal.modules.clinical")`.
```

It's missing closing double quote for all of them.